sources/org.osbuild.skopeo

```diff
@@ -103,13 +107,21 @@ class SkopeoSource(sources.SourceService):
         digest = image["digest"]
         tls_verify = image.get("tls-verify", True)
         transport = image.get("containers-transport", DOCKER_TRANSPORT)
+        location = image.get("storage-location", "")
 
         with tempfile.TemporaryDirectory(prefix="tmp-download-", dir=self.cache) as tmpdir:
             archive_dir = os.path.join(tmpdir, "container-archive")
             os.makedirs(archive_dir)
             os.chmod(archive_dir, 0o755)
 
-            reference = f"{imagename}@{digest}"
+            # If a storage location has been provided, create the specifier with
+            # the `overlay` driver. Leave the specifier blank if no source storage 
+            # path is provided for the container is being copied. Skopeo will read 
+            # the default storage path from the `/etc/containers/storage.conf` file 
+            # via  the `containers/storage` library, see: 
+            # https://github.com/containers/storage/blob/main/types/options.go#L53
```

Comment:
Can you replace this with a permalink that includes the commit ID?  Linking to `main` means we would link to a different line when the file is modified.